### DISCUSSION TITLE
GitHub student developer pack

### DISCUSSION THREAD

**Discussion by lavnishhh** [community/community#37466] [GitHub Education] [answered]
The github student dev pack requires me to upload my university's enrollment letter. However, the upload form only takes up images. Would it be fine if I use a screenshot of the enrollment lettter?

In case you are asking why is the enrollment letter a pdf, the time of my enrollment was the covid era, hence all of the process was online.

Can I upload a pdf screenshot?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Lavnishhh! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by lavnishhh** [reply]
I couldn't figure out the right category so I chose this. Thank you so much :)

**Comment by Omobayode6** [comment, marked as answer]
You can screenshot the pdf and upload the screenshot image.

I use that for mine and it works.

**Reply by ghost** [reply]
how do u attach pic?my browser n phone cant do that

**Reply by lavnishhh** [reply]
> how do u attach pic?my browser n phone cant do that

Take a screenshot of your pdf and upload it?

**Comment by Musa-ulug** [comment]
![Ekran Görüntüsü (7)](https://user-images.githubusercontent.com/117463866/200336580-441b36ba-a2f4-409a-afe1-d56a82604fc7.png)

I upload the evidence to the system but it doesn't work, the screen stays like this

**Comment by Musa-ulug** [comment]
![Ekran Görüntüsü (9)](https://user-images.githubusercontent.com/117463866/200337176-c8e41ffb-4a22-4fa2-ac5b-e5c5197c2a56.png)


I upload the evidence to the system but it doesn't work, the screen stays like this

**Reply by ScyllaHide** [reply]
> ![Ekran Görüntüsü (9)](https://user-images.githubusercontent.com/117463866/200337176-c8e41ffb-4a22-4fa2-ac5b-e5c5197c2a56.png)
> 
> I upload the evidence to the system but it doesn't work, the screen stays like this

same problem, i uploaded an image of the actual pdf (universities use pdf files these days and no images!!!)

Its stuck for 10 mins now, any ideas if a person can look at it? Like it used to be in 2016, 2018 and 2020 as it worked without fancy image recognition 🙄 

Or is there a support i can write to?

**Reply by ScyllaHide** [reply]
Weird just wanted to write a support ticket, tried again and it worked finally. Looks like it needs a few tries and there should be enough headroom for download/upload.

**Comment by glwbr** [comment]
Hello, I'm also having problems while trying to upload my enrollment letter, I tried to upload a picture from my webcam but the site gets stuck. Antyone else have an idea about how to solve this ?

**Comment by Naiyavyas1231** [comment]
Hello, I Wanted to ask you that I have just Created Account and got Approved for Student Developer Pack, but the problem is I didn't receive "Canva Pro" in the Student Developer Pack. Pls tell me Why I have not received Canva pro and how can I get it?, its very Important for me so...

CAN ANYONE HELP ME TO GET CANVA PRO... I NEED IT , ITS V.IMPORTANT

**Comment by akasyaotomotiv** [comment]
hello, I bought the pro package but I can also buy the student package

12 Kas 2022 Cmt 19:10 tarihinde Naiyavyas1231 ***@***.***>
şunu yazdı:

**Comment by jaekerrclayton** [comment]
Hello, I was given an activation link to activate my student developer pack through my school, but the link is currently not working and returns a 404. I have contacted my school administrator, they were unable to help and recommended contacting you instead. Would appreciate any insight or assistance, if possible! Thank you.

**Comment by theharshjha** [comment]
I don't have a college email. then how should I apply for a student pack?

**Comment by asdrascheper** [comment]
For me, it makes me to take a picture with my webcam or smartphone, which doesn't make any sense. I'm using OBS to fake a screenshot as a webcam because smartphones don't take good pictures of documents. Plus, everything is in pt-br, so I have to translate the entire document in a notepad window just to make a screenshot, put it in OBS, use its virtual webcam and take another screenshot. 
This application process is ridiculous. Who invented that? Seriously, I lost like an hour trying to make something out of this and it still says I have to update my profile and check my location because I'm a distant learner.
